fusor-ember-cli/app/components/base-f.js

```diff
@@ -16,11 +16,11 @@ export default Ember.Component.extend({
   },
 
   labelClassSize: Ember.computed(function () {
-    return this.getWithDefault('labelSize', 'col-lg-2 col-md-3 col-sm-5');
+    return this.getWithDefault('labelSize', 'col-lg-3 col-md-3 col-sm-4 col-xs-4 col-xs-offset-1');
```

Comment:
This seems like a rather large change to put in the default class for base-f (going from overall width of 12 to 9 in the case of small width windows).  If only the naming-scheme-modal was incorrect, shouldn't we just modify the labelSize and inputSize on that modal instead of changing the defaults for base-f and affecting all input fields that use base-f?